websocketOutportDriver/data/operations.go

```diff
@@ -1,12 +1,20 @@
 package data
 
+// WebSocketConfig holds the configuration needed for instantiating a new web socket server
 type WebSocketConfig struct {
 	URL             string
 	WithAcknowledge bool
 }
 
-type OperationType uint16
+// OperationType defines the type to be used to group web socket operations
+type OperationType uint8
 
+// OperationTypeFromUint64 returns the operation type based on the provided uint64 value
+func OperationTypeFromUint64(value uint64) OperationType {
```

Comment:
or simply: `func NewOperationType(value uint64) OperationType`

Reply:
No, as we use `New..` for constructing objects, I'd keep the naming, as `OperationType` is just a type, not a struct